crates/sui-graphql-rpc/src/mutation.rs

```diff
@@ -12,6 +12,7 @@ use sui_types::{signature::GenericSignature, transaction::Transaction};
 
 pub struct Mutation;
 
+/// Mutations are used to write to the Sui network.
```

Comment:
i dont know if this would be useful

Reply:
🤷  If we are currently showing a "No description" instead of this, then it might be marginally better, but otherwise, yes I agree.